content/java/concepts/math-methods/terms/floor/floor.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'Floor'
+Description: 'Returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags: 
+  - 'Documentation'
+CatalogContent: 
+  - 'learn-java/'
+  - 'paths/computer-science'
+---
+
+The **`.floor()`** method included in the `Math` class returns the largest integer value that is less than or equal to the argument.
+
+## Syntax
+
+When the provided value is either an integer, zero, not a number, or positive or negative infinity.
```

Comment:
I'm not sure what you are trying to say here.

Reply:
I removed this line as it was already stated further down in the Syntax section.